pkgs/development/python-modules/virtual-glob/default.nix

```
@@ -0,0 +1,50 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  lib,
+  pytest,
+  pytestCheckHook,
+  pythonAtLeast,
+}:
+
+buildPythonPackage rec {
+  pname = "virtual-glob";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "chrisjsewell";
+    repo = "virtual-glob";
+    tag = "v${version}";
+    hash = "sha256-ocCa8m7mPPvzOZHPrraSEdSJZwRJoYO/Q7nyDbhIFu8=";
+  };
+
+  build-system = [
+    flit-core
+  ];
+
+  optional-dependencies = {
+    testing = [
+      pytest
+    ];
+  };
+
+  pythonImportsCheck = [
+    "virtual_glob"
+  ];
+
+  disabledTests = lib.optionals (pythonAtLeast "3.13.1") [
+    "test_baseline_pathlib[a/b/c/d.txt;**]"
+    "test_baseline_pathlib[a/b/c/d.txt;a/b/**]"
```

```suggestion
    "test_baseline_pathlib"
```
Think our current disabledTests setup does not like parametrized tests